rotkehlchen/assets/spam_assets.py

```diff
@@ -79,12 +79,18 @@ def update_spam_assets(db: 'DBHandler', assets_info: list[dict[str, Any]]) -> in
     # and in that case query_token_spam_list add it to the global DB
     with db.conn.read_ctx() as cursor:
         ignored_asset_ids = db.get_ignored_asset_ids(cursor)
-    assets_added = 0
+
+    write_data = []
     for token in spam_tokens:
         if token.identifier in ignored_asset_ids:
             continue
 
-        with db.user_write() as write_cursor:
-            db.add_to_ignored_assets(write_cursor=write_cursor, asset=token)
-        assets_added += 1
+        write_data.append(('ignored_asset', token.identifier))
```

Comment:
Maybe is faster here to use list comprhension?

Reply:
Maybe. I did not do it since I kept `assets_added` at first, but then thought of `cursor.rowcount` for it. Will see if it makes sense and add in next iteration/PR